pkg/service/lb.go

```diff
@@ -106,23 +106,57 @@ func lbTags(app string, process string) map[string]string {
 	}
 }
 
+type loadBalancerExposureError struct {
```

Comment:
Prefixing these with `loadBalancer` feels a little verbose since we're already in package lb. How about just `ExposureError` (exported so consumers can type assert on it). e.g.

```go
if err, ok := err.(*lb.ExposureError); ok {
        // Stuff
}
```